fdbclient/include/fdbclient/Tenant.h

```diff
@@ -37,12 +37,14 @@ struct TenantMapEntry {
 
 	int64_t id;
 	Key prefix;
+	// indicates whether the current tenant entry is encrypted
```

Comment:
Encryption is property of the tenant, also, reading the comment the impression I got is: "TenantEntry" is encrypted?:)